### ISSUE TITLE
Error when generate Jhipter and Quarkus in MacOS

### ISSUE THREAD

**Issue by Vocaotrinh1999nlu** [jhipster/generator-jhipster-quarkus#298]
When I generate jhipter with Quarkus, I following in official document but got some error, please help me resolve that
**Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './cli/utils' is not defined by "exports" in /opt/homebrew/lib/node_modules/generator-jhipster-quarkus/node_modules/generator-jhipster/package.json
    at new NodeError (node:internal/errors:406:5)
    at exportsNotFound (node:internal/modules/esm/resolve:268:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:598:9)
    at resolveExports (node:internal/modules/cjs/loader:547:36)
    at Module._findPath (node:internal/modules/cjs/loader:621:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1034:27)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at Object.<anonymous> (/opt/homebrew/lib/node_modules/generator-jhipster-quarkus/cli/jhipster-quarkus.js:22:20) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}**

**Comment by mraible** [comment]
Can you please try with Node 16 and let us know if that works?

**Comment by djibythiaw** [comment]
Hello,
I have the same issue with node v20.9.0 and also node v18.18.2
but it's working fine with node v16.20.2

**Comment by ben-hua** [comment]
> Can you please try with Node 16 and let us know if that works?

I found `in develop` has been removed in the readMe. When will this main version be released?

**Comment by skin27** [comment]
I have the same issue on Windows 11 with node Node.js v18.16.0

```
D:\intellij\workspace\qhipster81>jhipster-quarkus
node:internal/modules/cjs/loader:571
      throw e;
      ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './cli/utils' is not defined by "exports" in C:\Users\Raymond\AppData\Roaming\npm\node_modules\generator-jhipster-quarkus\node_modules\generator-jhipster\package.json
    at new NodeError (node:internal/errors:399:5)
    at exportsNotFound (node:internal/modules/esm/resolve:361:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:697:9)
    at resolveExports (node:internal/modules/cjs/loader:565:36)
    at Module._findPath (node:internal/modules/cjs/loader:634:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1061:27)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (C:\Users\Raymond\AppData\Roaming\npm\node_modules\generator-jhipster-quarkus\cli\jhipster-quarkus.js:22:20) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

```

Using Jhipster 8.1.0

**Comment by thanasis812** [comment]
Error still present using Windows 11 and node 20.11.0. Any plans on upgrading to support  20.11.0 any time soon?

**Comment by vishal423** [comment]
it should be fixed with 3.0.0 release
